test/EmailAddressTest.php

```diff
@@ -741,8 +741,8 @@ public function testUseMxCheckBasicValid()
         ];
 
         if (extension_loaded('intl')) {
-            $emailAddresses[] = 'иван@письмо.рф';
-            $emailAddresses[] = 'xn--@-7sbfxdyelgv5j.xn--p1ai';
+            $emailAddresses[] = 'test@письмо.рф';
+            $emailAddresses[] = 'test@xn--h1aigbl0e.xn--p1ai';
```

Comment:
Because
```
var_dump(
    idn_to_ascii('иван@письмо.рф'),
    'test@' . idn_to_ascii('письмо.рф')
);
```
prints
```
string 'xn--@-7sbfxdyelgv5j.xn--p1ai' (length=28)

string 'test@xn--h1aigbl0e.xn--p1ai' (length=27)
```